### ISSUE TITLE
Cms module collections missing event prefix

### ISSUE THREAD

**Issue by extroniks** [magento/magento2#9900]
### Preconditions

1. Magento 2.1.7 CE

### Steps to reproduce

1. Try to observe events: cms_page_collection_load_before, cms_page_collection_load_after, cms_block_collection_load_before, cms_block_collection_load_after

### Expected result

1. Event is dispatched and listeners execute

### Actual result

1. Listeners aren't executed because the collections are missing the event prefix

Code lines to reproduce:
```
/* @var \Magento\Cms\Model\ResourceModel\Page\Collection $collection */
$collection = $this->_objectManager
    ->create(\Magento\Cms\Model\ResourceModel\Page\Collection::class);
$collection->load();

/* @var \Magento\Cms\Model\ResourceModel\Block\Collection $collection */
$collection = $this->_objectManager
    ->create(\Magento\Cms\Model\ResourceModel\Block\Collection::class);
$collection->load();
```

**Comment by magento-engcom-team** [comment]
We updated issue description:

> Code lines to reproduce:
> ```
> /* @var \Magento\Cms\Model\ResourceModel\Page\Collection $collection */
> $collection = $this->_objectManager
>     ->create(\Magento\Cms\Model\ResourceModel\Page\Collection::class);
> $collection->load();
> 
> /* @var \Magento\Cms\Model\ResourceModel\Block\Collection $collection */
> $collection = $this->_objectManager
>     ->create(\Magento\Cms\Model\ResourceModel\Block\Collection::class);
> $collection->load();
> ```

**Comment by magento-engcom-team** [comment]
@StefanErakovic, thank you for your report.
We've created internal ticket(s) MAGETWO-77434 to track progress on the issue.

**Comment by okorshenko** [comment]
we need to backport this into 2.2-develop

**Comment by okorshenko** [comment]
The issue needs to be assigned to @convenient

**Comment by okorshenko** [comment]
@convenient is working on it #SQUASHTOBERFEST

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-80885

**Comment by okorshenko** [comment]
@StefanErakovic thank you for your report. The issue is already fixed in 2.2-develop branch (PR: #11223  by @convenient ) and will be available in the Magento 2.2.2 release
